.storybook/preview.js

```diff
@@ -85,6 +85,11 @@ export const parameters = {
 				color: "#fff",
 				description: "Should not be used and will not receive updates.",
 			},
+			migrated: {
+				background: "rgb(84, 36, 219)",
```

Comment:
I have the rgb for indigo-1100 in light mode. We could forseeably use tokens for the background & font color, but I saw we _didn't_ do that with the deprecated status, so I elected not to do that for migrated.

Reply:
Yeah I think I tried once and there was a type-check in place for the tool so it throws an error.